config/schedule.rb

```diff
@@ -32,6 +32,7 @@
 
 # every night around 5 AM
 every nightly_cron_str do
+  rake 'flag_overdue'
```

Comment:
Did we forget to add the `flag_missed` task here? Also, make sure to add these tasks to the Heroku bucket tasks as well.